hnn_core/dipole.py

```diff
@@ -37,6 +37,8 @@ def simulate_dipole(net, tstop, dt=0.025, n_trials=None, record_vsec=False,
     record_isec : 'all' | 'soma' | False
         Option to record voltages from all sections ('all'), or just
         the soma ('soma'). Default: False.
+    record_dcell : bool
+        Option to record dipole from individual cells. Default: False.
```

Comment:
Is there a computational hit when you do this? I think users should be warned ...

Reply:
If there is, it is substantially less than `record_isec` and `record_vsec` because the individual cell dipoles have to be recorded by default. The only difference here is that we save them at the end instead of just adding them together.

Reply:
With that said I personally haven't noticed a performance hit, the main concern would be taking up too much RAM if the recording is really long